app/qml/main.qml

```diff
@@ -402,7 +402,23 @@ ApplicationWindow {
     Connections {
         target: __loader
         onLoadingStarted: projectLoadingScreen.visible = true
-        onLoadingFinished: projectLoadingScreen.visible = false
+        onLoadingFinished:
+        {
+            projectLoadingScreen.visible = false;
+            if ( invalidLayers.length > 0 )
+            {
+                let warningMessage = qsTr( "WARNING: the following layers are invalid: " );
+                for ( let i = 0; i < invalidLayers.length - 1; ++i )
+                {
+                    warningMessage += qsTr( "'%1' " ).arg( invalidLayers[i] );
+                }
+                if ( invalidLayers.length > 1 )
+                    warningMessage += qsTr( "and '%1' " ).arg( invalidLayers[invalidLayers.length - 1] );
```

Comment:
Similar to previous point. I am afraid when translators should add special characters

```suggestion
                    warningMessage += qsTr( "and" ) + " %1".arg( invalidLayers[invalidLayers.length - 1] );
```